### ISSUE TITLE
Typo in Getting Started?

### ISSUE THREAD

**Issue by bkellerman** [blei-lab/edward#863]
When trying to run the getting started code.

```
  File "test1.py", line 20, in <module>
    qW_0 = Normal(loc=tf.get_variable("qW_0/loc", [D, 4]),
NameError: name 'D' is not defined
```

It appears there is a typo.

```
qW_0 = Normal(loc=tf.get_variable("qW_0/loc", [D, 2]),
              scale=tf.nn.softplus(tf.get_variable("qW_0/scale", [D, 2])))
```

Should be.

```
qW_0 = Normal(loc=tf.get_variable("qW_0/loc", [1, 2]),
              scale=tf.nn.softplus(tf.get_variable("qW_0/scale", [1, 2])))
```

**Comment by dustinvtran** [comment]
Good catch. D is defined in the notebook but not the getting started webpage. Can you hard-code it to 1 in the webpage? (`docs/tex/`) Contributions welcome.

**Comment by dustinvtran** [comment]
Fixed in #864.
